packages/kbn-shared-ux-components/src/page_template/solution_nav/solution_nav.tsx

```diff
@@ -58,6 +58,10 @@ export type KibanaPageTemplateSolutionNavProps = Omit<
    */
   isOpenOnDesktop?: boolean;
   onCollapse?: () => void;
+  /*
+   * Force the open state on desktop
+   */
+  isForceOpenOnDesktop?: boolean;
```

Comment:
@sebelga `canBeCollapsed` is imho not the exact naming because we can have a situation when the side nav has been collapsed but then for example a tour has been activated. So that should force the side nav to open (expand) for the tour to display correctly. And the side nav should not be able to close (collapse) during the tour.

Reply:
Based on your explanation in both comments, I think I actually agree with @sebelga. The main thing I'd like to avoid is having props that are created for very specific situations, should those ever change then the prop is essentially tech dept. The way I like to think about these types of situations is to just make the component itself more flexible in state, meaning, allow more control over how it behaves directly by the consumer.

I propose changing this prop name and what it does to:

- `canBeCollapsed`: Explicitly states that this collapsible functionality (button and all) should or should not be allowed. The default would be `true`, but when `false` since the functionality has been turned off, the sidebar should be forced into the open state and the collapse button hidden.

If we wanted to get fancy with it, we could allow the type to be `boolean | EuiBreakpoint[]` in which you could customize at which array of breakpoints the collapse functionality is allowed. This would ensure that smaller windows can still have the flyout or accordion collapsed.

Sorry that I'm being nit-picky here with terminology, but since this is a widely used component, I want to make sure it's especially clear what the props do and yet generic enough that it can apply to multiple situations.

Reply:
thanks a lot, @cchaos! I appreciate your input 👍  
I'll ping you once I implemented your suggestions